client/src/components/User/UserBeaconSettings.vue

```diff
@@ -221,7 +221,7 @@ export default {
       try {
         await axios.get(`${this.root}api/users/${this.userId}/beacon`).then(
             response => {
-              this.enabled = response.data.enabled == true;
+              this.enabled = response.data.enabled == true || response.data.enabled == "true";
```

Comment:
this is both the same @Paprikant

Reply:
You must ensure your API endpoint returns proper JSON (`true` or `false`). FastAPI and Pydantic would have prevented that ;)
Once you do that, it will be simply:
```suggestion
              this.enabled = response.data.enabled;
```

Reply:
I am still warming up to python :smiling_face_with_tear: 

The `string_as_bool` function does exactly what we need here, thanks for pointing that out :+1: 
```python
def string_as_bool(string: typing.Any) -> bool:
    if str(string).lower() in ("true", "yes", "on", "1"):
        return True
    else:
        return False
```

I am doing this after a long day but I am still pretty confident that this time whatever the database throws at us will be cast to the correct boolean :slightly_smiling_face: